app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
```

Comment:
**[REQUIRED]**
* User should have many comments.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/user_has_many_comments.png) from Odin Project).

Reply:
We actually have many comments:

`class User < ApplicationRecord
  has_many :posts, dependent: :delete_all
  validates :name, presence: true
  validates :email, presence: true, uniqueness: true
  validates_format_of :email, with: /\A[^@\s]+@([^@\s]+\.)+[^@\s]+\z/
end`